Sources/swift-package-manager/SwiftPM.swift

```diff
@@ -21,8 +21,9 @@ let execName = (try? AbsolutePath(validating: firstArg).basenameWithoutExt) ??
     (try? RelativePath(validating: firstArg).basenameWithoutExt)
 
 @main
+@available(macOS 10.15, macCatalyst 13, iOS 13, tvOS 13, watchOS 6, *)
```

Comment:
Not sure why this is needed, we already have a macOS 10.15.4 and iOS 13.4 minimum deployment target?

Reply:
I think we ran into issues with CI if not set. happy to remove this when we bump to macOS 12?